invertedai/large/common.py

```diff
@@ -0,0 +1,153 @@
+from typing import List, Optional, Tuple
+from pydantic import BaseModel
+
+from invertedai.common import AgentAttributes, AgentState, RecurrentState, Point
+
+class Region(BaseModel):
+    """
+    A data structure to contain information about a region in a map to be used in large simulations.
+
+    See Also
+    --------
+    AgentAttributes
+    """
+
+    center: Optional[Point] = None # The center of the region if such a concept is relevant (e.g. center of a square, center of a rectangle)
+    fov: Optional[float] = None # Side length of the region for the default interpretation of a region as a square
```

Comment:
`size` or `length` would be a better name, as the region definition doesn't necessarily have anything to do with viewing.

Reply:
Done.